test/Test/Database/LSMTree/Normal/StateMachine.hs

```diff
@@ -1606,22 +1606,12 @@ updateStats action lookUp modelBefore _modelAfter result =
 
 -- | Tags for every step
 data Tag =
-    -- | Snapshot with a name that already exists
-    SnapshotTwice
     -- | Open an existing snapshot
-  | OpenExistingSnapshot
-    -- | Open a missing snapshot
-  | OpenMissingSnapshot
+    OpenExistingSnapshot
     -- | Delete an existing snapshot
   | DeleteExistingSnapshot
-    -- | Delete a missing snapshot
-  | DeleteMissingSnapshot
-    -- | Open a snapshot with the wrong label
-  | OpenSnapshotWrongLabel -- TODO: implement
     -- | A merge happened on level @n@
   | MergeOnLevel Int -- TODO: implement
-    -- | A table was closed twice
-  | TableCloseTwice String -- TODO: implement
   deriving stock (Show, Eq, Ord)
```

Comment:
Again, one could argue that we can keep these tags

Reply:
Why, if they can never happen? It makes it look like they ought to happen?

Or do you mean if we were to do DL style unit tests instead?